tests/providers/neo4j/operators/test_neo4j.py

```diff
@@ -46,16 +28,13 @@
 
 
 class TestNeo4jOperator(unittest.TestCase):
-    def setUp(self):
-        args = {'owner': 'airflow', 'start_date': DEFAULT_DATE}
-        dag = DAG(TEST_DAG_ID, default_args=args)
-        self.dag = dag
-
-    @mock.patch('airflow.providers.neo4j.operators.neo4j.Neo4jOperator.get_hook')
+    @mock.patch('airflow.providers.neo4j.operators.neo4j.Neo4jHook')
     def test_neo4j_operator_test(self, mock_hook):
 
         sql = """
             MATCH (tom {name: "Tom Hanks"}) RETURN tom
             """
-        op = Neo4jOperator(task_id='basic_neo4j', sql=sql, dag=self.dag)
-        op.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
```

Comment:
This test took an unnecessarily long time - 6 seconds. We need a full run, but running the execute method is enough.